apps/web/components/dashboard/bookmarks/EditorCard.tsx

```diff
@@ -139,31 +138,33 @@ export default function EditorCard({ className }: { className?: string }) {
     }
   };
 
+  const OS = getOS();
+
   return (
     <Form {...form}>
       <form
         className={cn(
           className,
-          "flex flex-col gap-2 rounded-xl bg-card p-4",
+          "relative flex flex-col gap-2 rounded-xl bg-card p-4",
           cardHeight,
         )}
         onSubmit={form.handleSubmit(onSubmit, onError)}
       >
-        <div className="flex justify-between">
-          <p className="text-sm">NEW ITEM</p>
+        <div
+          className={`absolute right-2 top-2 ${!inputRef.current?.value ? "block" : "hidden"} `}
```

Comment:
Should we be using `cn` here instead of string interpolation?

Reply:
resolved